Jenkinsfile

```diff
@@ -1,24 +1,40 @@
 pipeline {
   agent { label 'subman' }
   options {
-    timeout(time: 10, unit: 'MINUTES')
+    timeout(time: 15, unit: 'MINUTES')
+  }
+  environment {
+    REGISTRY_URL = 'quay.io/candlepin'
+    GIT_HASH = sh(returnStdout: true, script: "git rev-parse HEAD").trim()
```

Comment:
Pull the GIT_HASH at the beginning to pass it to every step (mostly used for container tag).  This is because each stage will try to do its own merge commit from main to the upstream PR.  I'm not sure how to turn this off without adding a bunch of complexity to the stages... need to research.